packages/react-dom/src/events/__tests__/SyntheticClipboardEvent-test.js

```diff
@@ -35,42 +47,119 @@ describe('SyntheticClipboardEvent', () => {
             items: null,
             types: null,
           };
-          var clipboardEvent = createEvent({clipboardData: clipboardData});
 
-          expect(clipboardEvent.clipboardData).toBe(clipboardData);
+          var eventHandler = event => {
+            expect(event.clipboardData).toBe(clipboardData);
+            expectedCount -= 1;
+          };
+
+          var container = document.createElement('div');
+          var div = ReactDOM.render(
+            <div
+              onCopy={eventHandler}
+              onCut={eventHandler}
+              onPaste={eventHandler}
+            />,
+            container,
+          );
+          document.body.appendChild(div);
```

Comment:
Shouldn't we add the `container` to body instead? This detaches the div from the original container.